code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -19,8 +19,8 @@ Bonus
 /datum/symptom/sneeze
 
 	name = "Sneezing"
-	stealth = -2
-	resistance = 3
+	stealth = -1

```

Comment:
Why are you decreasing stealth rating? Sneezing symptom is a useful tool for revealing stealth virus mixtures in the PANDEMIC. If you are doing this to make it more useful in combat viruses, please provide a suitable roundstart replacement. Perhaps give -2 stealth to cough instead?